docs/rules/no-missing-label-refs.md

```diff
@@ -30,6 +30,24 @@ Examples of **incorrect** code for this rule:
 [eslint]
 ```
 
+## Options
+
+The following options are available on this rule:
+
+* `allowLabels: Array<string>` - labels to allow when checking for missing label references. (default: `[]`)
```

Comment:
In your original issue you mentioned wanting to use `ignoreLabels`. Why the change?

Reply:
`allowLabels` amkes sense to me 👍🏻

Reply:
Ah missed, that comment, `allowLabels` is fine with me.